python/version_details/shotgun_entities/card_widget.py

```diff
@@ -462,3 +457,69 @@ def field_widgets(self):
     show_labels = property(_get_show_labels, _set_show_labels)
 
 
+class _OrderedDict(object):
+    """
+    An OrderedDict-like class. This is implemented here in order to maintain
+    backwards compatibility with pre-2.7 releases of Python.
+    """
+    def __init__(self, **kwargs):
+        """
+        Constructor. Emulates the behavior of the dict() type.
+        """
```

Comment:
might be worth pointing out here that the order of the supplied dict will not be maintained.